react/src/base/inputs/components/SprkInputElement/SprkInputElement.js

```diff
@@ -51,6 +52,18 @@ class SprkInputElement extends Component {
       }
     };
 
+    const getAriaDescribedBy = () => {
```

Comment:
nice! this is definitely something we should be incorporating to our other components that use aria-describedby